probe-rs/src/bin/probe-rs/cmd/info.rs

```diff
@@ -320,14 +321,10 @@ fn show_arm_info(interface: &mut dyn ArmProbeInterface, dp: DpAddress) -> Result
             }
 
             ApInformation::Other { address, idr } => {
-                let designer = idr.DESIGNER;
+                let jep = idr.DESIGNER;
 
-                let cc = (designer >> 7) as u8;
-                let id = (designer & 0x7f) as u8;
-
-                let jep = jep106::JEP106Code::new(cc, id);
-
-                let ap_type = if designer == 0x43b {
+                let jep_arm = JEP106Code::new(4, 0x3b);
```

Comment:
Can we maybe put a unit test that this indeed resolves to ARM? :) could put it in a const :)

Reply:
Does the last change fit better ? :)